coordinator/src/routes/health/get-status.ts

```diff
@@ -34,28 +59,22 @@ export default [
     path: "/_status",
     handler: async (request: Hapi.Request, h: Hapi.ResponseToolkit): Promise<Hapi.ResponseObject> => {
       const checks: { [name: string]: Array<StatusCheckResponse> } = {
```

Comment:
Maybe inline `checks` here and on line 73, then we won't need the ugly `{ [name: string]: Array<StatusCheckResponse> }` type definition?

Reply:
done